crates/chia-bls/src/cached_bls.rs

```diff
@@ -1,253 +1,260 @@
-// This cache is a bit weird because it's trying to account for validating
-// mempool signatures versus block signatures. When validating block signatures,
-// there's not much point in caching the pairings because we're probably not going
-// to see them again unless there's a reorg. However, a spend in the mempool
-// is likely to reappear in a block later, so we can save having to do the pairing
-// again. So caching is primarily useful when synced and monitoring the mempool in real-time.
-
-use crate::aggregate_verify_gt as agg_ver_gt;
-use crate::gtelement::GTElement;
-use crate::hash_to_g2;
-use crate::PublicKey;
-use crate::Signature;
-use lru::LruCache;
-
-use sha2::{Digest, Sha256};
 use std::borrow::Borrow;
 use std::num::NonZeroUsize;
 
-#[cfg(feature = "py-bindings")]
-use pyo3::exceptions::PyValueError;
-#[cfg(feature = "py-bindings")]
-use pyo3::pybacked::PyBackedBytes;
-#[cfg(feature = "py-bindings")]
-use pyo3::types::{PyAnyMethods, PyList};
-#[cfg(feature = "py-bindings")]
-use pyo3::{pyclass, pymethods, PyResult};
+use lru::LruCache;
+use sha2::{Digest, Sha256};
 
-#[cfg_attr(feature = "py-bindings", pyclass(name = "BLSCache"))]
-#[derive(Clone)]
-pub struct BLSCache {
-    cache: LruCache<[u8; 32], GTElement>, // LRUCache of hash(pubkey + message) -> GTElement
+use crate::{aggregate_verify_gt, hash_to_g2};
+use crate::{GTElement, PublicKey, Signature};
+
+/// This cache is a bit weird because it's trying to account for validating
+/// mempool signatures versus block signatures. When validating block signatures,
+/// there's not much point in caching the pairings because we're probably not going
+/// to see them again unless there's a reorg. However, a spend in the mempool
+/// is likely to reappear in a block later, so we can save having to do the pairing
+/// again. So caching is primarily useful when synced and monitoring the mempool in real-time.
+#[cfg_attr(feature = "py-bindings", pyo3::pyclass(name = "BLSCache"))]
+#[derive(Debug, Clone)]
+pub struct BlsCache {
```

Comment:
Renamed this to `BlsCache` to follow [the Rust convention](https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case).

Reply:
I disagree with this because BLS is an acronym, not a word

Reply:
This is what the convention says though:
> In UpperCamelCase, acronyms and contractions of compound words count as one word: use Uuid rather than UUID, Usize rather than USize or Stdin rather than StdIn. In snake_case, acronyms and contractions are lower-cased: is_xid_start.

Reply:
Other conventions seem to include this as well (for acronyms longer than 2 characters at least):
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers